### ISSUE TITLE
JDK9 warnings on start

### ISSUE THREAD

**Issue by hamnis** [sbt/sbt#3569]
## steps
`sbt update`

## problem

```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by sbt.internal.librarymanagement.ivyint.ErrorMessageAuthenticator$ (file:/Users/maedhros/.sbt/boot/scala-2.12.3/org.scala-sbt/sbt/1.0.1/librarymanagement-ivy_2.12-1.0.0.jar) to field java.net.Authenticator.theAuthenticator
WARNING: Please consider reporting this to the maintainers of sbt.internal.librarymanagement.ivyint.ErrorMessageAuthenticator$
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

## expectation
No warnings

## notes

sbt version: 1.0.1

**Comment by dwijnand** [comment]
Duplicate of https://github.com/sbt/librarymanagement/issues/169.

**Comment by eed3si9n** [comment]
Re-opening this to track on sbt/sbt. Any discussions should take place in sbt/librarymanagement#169.

**Comment by eed3si9n** [comment]
I merged lm, but didn't incorporate fix for sbt/launcher#38 in 1.0.3.

**Comment by mkurz** [comment]
This is still in the release notes of 1.0.3. See the last point of "Bug fixes" here: https://github.com/sbt/sbt/releases/tag/v1.0.3

**Comment by dwijnand** [comment]
This issue, the one about "Illegal reflective access by sbt.internal.librarymanagement.ivyint.ErrorMessageAuthenticator$", is fixed and shipped in 1.0.3, and we should close it.

https://github.com/sbt/launcher/issues/38 is also fixed, but sadly didn't ship in 1.0.3. We'll track shipping it in 1.0.4 with https://github.com/sbt/sbt/issues/3676.
